test/support/commands.js

```diff
@@ -1,3 +1,4 @@
 Cypress.Commands.add('getByTestId', testId => cy.get(`[data-testid="${testId}"]`));
 Cypress.Commands.add('getByTitle', title => cy.get(`[title="${title}"]`));
 Cypress.Commands.add('getTooltip', () => cy.get('[role="tooltip"]'));
+Cypress.Commands.add('getByDataPreview', dataPreview => cy.get(`[data-preview="${dataPreview}"]`));
```

Comment:
We should try to avoid selecting on custom Styleguidist properties like this in case we want to migrate to a different framework. Navigating to the page should be enough to ensure that the test is executing on the correct element.